spec/models/like_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe Like, type: :model do
+  subject { Like.new(user_id: 1, post_id: 3) }
+  # replace user_id and post_id with valid user and post ids
+
+  context 'When testing likes' do
+    it 'Should increment the likes counter of a post' do
+      post = Post.find(subject.post_id)
+      likes_counter = post.comments_counter
+
+      subject.increment_likes_count
+      updated_post = Post.find(subject.post_id)
+      expect(updated_post.likes_counter).to eq(likes_counter + 1)
+    end
+  end
+end
```

Comment:
- [ ] When I run the `comment_spec` test file, some test cases fail

![image](https://github.com/BenM4k/BlogApp/assets/77106962/55ea9abf-d8f0-402e-9ba1-6b7e3d5bb847)

Kindly fix it so that you are sure that the validations and the methods for this model function properly 👍 😎

Reply:
@Gambit142 the test cases are working, of the correct user is in the test db

Reply:
I refactored not using static ids